tests/ImageSharp.Formats.Tiff.Tests/Formats/Tiff/TiffDecoderHeaderTests.cs

```diff
@@ -0,0 +1,119 @@
+// <copyright file="TiffDecoderHeaderTests.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Tests
+{
+    using System.IO;
+    using System.Linq;
+    using Xunit;
+
+    using ImageSharp.Formats;
+
+    public class TiffDecoderHeaderTests
+    {
+        public static object[][] IsLittleEndianValues = new[] { new object[] { false },
+                                                                new object[] { true } };
+
+        [Theory]
+        [MemberData(nameof(IsLittleEndianValues))]
+        public void ReadHeader_ReadsEndianness(bool isLittleEndian)
```

Comment:
Agreed, great stuff. The others really should be. We should add a chore tag for issues so we can do this 😝